spiketools/sim/occupancy.py

```diff
@@ -0,0 +1,84 @@
+"""Simulate occupancy."""
+
+import numpy as np
+
+###################################################################################################
+###################################################################################################
+
+def single_trial_occ(num_bins, min_occ, max_occ):
```

Comment:
for function names, generall all sim functions start with `sim_`.
For this it probably wants to be something like: `sim_single_trial_occupancy`